### PR TITLE
Add FS3 Volume Backup and Rebuild Features

### PR DESCRIPTION
**Pull request by GuohaoMa**
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -170,8 +171,11 @@ func Main(args []string) {
 
 	initConfigAndLog()
 	initUserConfig(sysconfig.GetSysConfig().StandAlone)
+	scheduler.SendDealScheduler()
+	scheduler.BackupScheduler()
+	scheduler.RebuildScheduler()
 
-	fmt.Println(config.GetUserConfig().Fs3VolumeAddress)
+	fmt.Println("Your FS3 Server is running successfully. Please copy and paste the url below to open in a browser")
```
Comment (zzhao-nbai) [inline cmd/main.go:178]: change to logs.getloger.info

File: logs/ledger_info.log [hunk 2]
```diff
@@ -0,0 +1,64 @@
+time="2021-11-10 11:48:10.751" level=info msg="https://api.filswan.com ~/minio-data  8589934592   " func=github.com/minio/minio/cmd.initUserConfig file="github.com/minio/minio/cmd/main.go:193"
```
Comment (zzhao-nbai) [inline logs/ledger_info.log:1]: log file should be remove

File: scheduler/autobid.go [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+	//close db
+	sqlDB, err := db.DB()
+	if err != nil {
+		logs.GetLogger().Error(err)
+		return err
+	}
+	defer sqlDB.Close()
+
+	var backupJobs []PsqlVolumeBackupJob
+	db.Find(&backupJobs)
```
Comment (zzhao-nbai) [inline scheduler/autobid.go:103]: handle error

File: scheduler/autobid.go [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+	for _, v := range tasks {
+		var backupJob PsqlVolumeBackupJob
+		db.Where("uuid=?", v[0].Uuid).First(&backupJob)
+		timestamp := strconv.FormatInt(time.Now().UTC().UnixNano()/1000, 10)
+		backupJob.Status = StatusBackupTaskRunning
+		backupJob.UpdatedOn = timestamp
+		backupJob.MinerId = v[0].MinerFid
+		backupJob.DealCid = v[0].DealCid
+		backupJob.Cost = v[0].Cost
+		db.Save(backupJob)
```
Comment (zzhao-nbai) [inline scheduler/autobid.go:167]: handle err

File: scheduler/rebuild.go [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+		logs.GetLogger().Error(err)
+		return err
+	}
+	return err
+}
+
+func GetOneRunningRebuildJob(db *gorm.DB) ([]PsqlVolumeRebuildJob, error) {
+	//get backupplans
+	var runningRebuildJob PsqlVolumeRebuildJob
+	db.Where("status=?", StatusRebuildTaskCreated).Or("status=?", StatusRebuildTaskRunning).First(&runningRebuildJob)
```
Comment (zzhao-nbai) [inline scheduler/rebuild.go:81]: handle error

File: cmd/main.go (hunk 1 above)
Comment (GuohaoMa) [reply cmd/main.go:178]: Done

File: logs/ledger_info.log (hunk 2 above)
Comment (GuohaoMa) [reply logs/ledger_info.log:1]: Removed

File: scheduler/autobid.go (hunk 3 above)
Comment (GuohaoMa) [reply scheduler/autobid.go:103]: Done

File: scheduler/autobid.go (hunk 4 above)
Comment (GuohaoMa) [reply scheduler/autobid.go:167]: Done

File: scheduler/rebuild.go (hunk 5 above)
Comment (GuohaoMa) [reply scheduler/rebuild.go:81]: Done

**Pull request merged**